web-api/src/storage/postgres.rs

```diff
@@ -968,9 +976,8 @@ impl storage::DocumentCandidate for Storage {
         &self,
         ids: impl IntoIterator<Item = &DocumentId>,
     ) -> Result<Warning<DocumentId>, Error> {
-        let (removed, mut failed) = self.postgres.remove_candidates(ids).await?;
-        failed.extend(self.elastic.delete_documents(&removed).await?);
-
+        let (removed, failed) = self.postgres.remove_candidates(ids).await?;
+        self.elastic.delete_by_parents(&removed).await?;
```

Comment:
:art: newline before return
```suggestion
        self.elastic.delete_by_parents(&removed).await?;

```